lib/index.js

```diff
@@ -275,7 +275,7 @@ const cherryPickPR = async (
   }
 };
 
-const PR_TITLE_REGEX = /bump ([\w-@\/]+) from ([\w\.-]+) to ([\w\.-]+)/i;
+const PR_TITLE_REGEX = /bump ([\w\.\-\@\/]+) from ([\w\.\-]+) to ([\w\.\-]+)/i;
```

Comment:
I don't think the escapes you added to the other things here are necessary
```suggestion
const PR_TITLE_REGEX = /bump ([\w\.-@\/]+) from ([\w\.-]+) to ([\w\.-]+)/i;
```